build.sbt

```diff
@@ -39,7 +39,10 @@ inThisBuild(Def.settings(
   onLoad in Global := {
     sLog.value.info(s"Building Akka HTTP ${version.value} against Akka ${AkkaDependency.akkaVersion}")
     (onLoad in Global).value
-  }
+  },
+  scalacOptions ++= Seq(
+    "-P:silencer:globalFilters=is deprecated \\(since 2\\.13\\." // remove when 2.12 support is dropped
```

Comment:
Hmm, I guess we do want to see this warning for things that are deprecated in 2.13 for which already in 2.12 there was an alternative available... but perhaps that's rare.

Reply:
Yeah, it seems rarer than the other way round currently.